test_unstructured/documents/structured_jsons/example.json

```diff
@@ -0,0 +1,121 @@
+[
+    {
+        "element_id": "897a8a47-377c-4ad6-aab8-39a929879537",
+        "metadata": {
+            "category_depth": 0,
+            "text_as_html": "<body class=\"Document\" id=\"897a8a47-377c-4ad6-aab8-39a929879537\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "3a6b156a-8176-4e17-be12-8264241f8136",
+        "metadata": {
+            "category_depth": 1,
+            "page_number": 1,
+            "parent_id": "897a8a47-377c-4ad6-aab8-39a929879537",
+            "text_as_html": "<div class=\"Page\" data-page-number=\"1\" id=\"3a6b156a-8176-4e17-be12-8264241f8136\" />"
+        },
+        "text": "",
+        "type": "UncategorizedText"
+    },
+    {
+        "element_id": "45b3d005-3468-484b-a1c7-b53998115412",
```

Comment:
we have done a lot of work to always format element_id's as a hash, which is pretty much a built in expectation.
e.g.:
``` 
 "element_id": "782cf07be8b3ab8f05188e479edb7f61",
 ```
. however, fortunately it appears there is a super quick fix, just remove the `-`'s in element_id (which still maps 1-1 with html `id`s)